WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuCoordinator.swift

```diff
@@ -22,26 +22,33 @@ final class HubMenuCoordinator: Coordinator {
 
     private let willPresentReviewDetailsFromPushNotification: () async -> Void
 
+    private let tapToPayBadgePromotionChecker: TapToPayBadgePromotionChecker
```

Comment:
Same comment as with `MainTabViewModel` regarding abstracting this class into a protocol, as would seem that we're leaking implementation details about the badges into the HubMenu navigation coordinator responsibility.